### PR TITLE
Awesome Books: Refactor

### PR DESCRIPTION
**Pull request by wale-prog**
### In this pull request my [partner](@Abdo9826 ) and I  collaborated to make some changes to the awesome book project as follows:
-  We have used JavaScript classes to split the code.
- We used objects and arrays, inside classes.
- We have Created class methods to add and remove books.
- There are no bugs after refactoring.
- We have added CSS styles to the application to make it match the design.

### CODE REVIEW

File: Books.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  addBooks(title, author) {
+    this.books.push({
+      title,
+      author,
+    });
+  }
+
+  removeBooks(author) {
+    this.author = author;
+    this.books = this.books.filter((book) => book.author !== author);
```
Comment (brytebee) [inline Books.js]: - [ ] **Required:** You are doing great so far. However, you are filtering a book to delete by the book's author. This creates a behaviour where if books have the same author, all of these books are deleted. Kindly fix this behaviour.

Delete duplicate

_**Hint**_ You could use an identifier separate from author or book title to solve this issue.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @wale-prog & @Abdo9826👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- You have used classes.✔️
- The app works well.✔️
- There are no linter errors.✔️
- There are no console errors.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Books.js (hunk 1 above)
Comment (wale-prog) [reply Books.js]: Thank you for the feedback, this has now been resolved by using the index position (number) of the remove button in the DOM to filter the local storage.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🍾🎉**
Hello @wale-prog and @Abdo9826 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**